css/includes/components/itilobject/_timeline.scss

```diff
@@ -99,7 +99,7 @@
     .timeline-content {
         position: relative;
         border-top-left-radius: 0;
-        max-width: 100%;
+        max-width: 700px;
```

Comment:
Not sure it will work in all window resolution contexts.

Reply:
I am just dubious about rendering on a small resolution, since 700px can be more than 100% of the container.

Reply:
There is no `width` or `min-width` defined so small containers are not impacted.

Reply:
I thought we may have an overflow, but I am probably wrong then.